### ISSUE TITLE
Properly blame gameoverlayrenderer64 in MiniDump

### ISSUE THREAD

**Issue by AvarianKnight** [citizenfx/fivem#2037]
`gameoverlayrenderer64.dll` leads to crashes that aren't fixable by Cfx and should properly blame steam overlay for the crash.

**Comment by saiy1337** [comment]
FiveM-Private (adhesive) hooks into GameOverlayRenderer64 which will give you a bad stack trace

**Comment by gottfriedleibniz** [comment]
Any issue with closing this Avarian?

**Comment by blattersturm** [comment]
Ideally one would be able to escalate crashes occurring here to Valve.

**Comment by gottfriedleibniz** [comment]
You are right. I was mixing this up with another [report](https://forum.cfx.re/t/issues-with-crash-reports-with-full-memory-dumps-enabled/5081146). This will definitively be left open.
